modules/conversion-center/base/widget-contact-button-base.php

```diff
@@ -295,6 +300,67 @@ private function add_top_bar_section(): void {
 		$this->end_controls_section();
 	}
 
+	private function add_message_bubble_section(): void {
+		$this->start_controls_section(
+			'message_bubble_section',
+			[
+				'label' => esc_html__( 'Message Bubble', 'elementor' ),
+				'tab'   => Controls_Manager::TAB_CONTENT,
+			]
+		);
+	
+		$this->add_control(
+			'message_bubble_name',
+			[
+				'label' => esc_html__( 'Name', 'elementor' ),
+				'type' => \Elementor\Controls_Manager::TEXT,
+				'default' => esc_html__( 'Rob', 'elementor' ),
+				'placeholder' => esc_html__( 'Type your name here', 'elementor' ),
+			]
+		);
+	
+		$this->add_control(
+			'message_bubble_body',
+			[
+				'label'       => esc_html__( 'Message', 'elementor' ),
+				'type'        => Controls_Manager::TEXTAREA,
```

Comment:
```suggestion
			[
				'label' => esc_html__( 'Message', 'elementor' ),
				'type' => Controls_Manager::TEXTAREA,
```
Please remove the spaces for all these in the file.

Reply:
done